clients/src/main/java/org/apache/kafka/clients/consumer/internals/MembershipManagerImpl.java

```diff
@@ -260,42 +743,16 @@ public Optional<String> serverAssignor() {
      * {@inheritDoc}
      */
     @Override
-    public ConsumerGroupHeartbeatResponseData.Assignment currentAssignment() {
+    public Set<TopicPartition> currentAssignment() {
         return this.currentAssignment;
     }
 
 
     /**
      * @return Assignment that the member received from the server but hasn't completely processed
-     * yet. Visible for testing.
```

Comment:
I made a comment up above about removing `targetAssignment()` from the `MembershipManager` interface because it was only used for testing. Does the removal of this statement imply that it will be used in non-testing later?

Reply:
Good catch, I needed it public at some point but it ended up not being needed in the end. So putting it back to package-private and "Visible for testing"